### ISSUE TITLE
Posted messages going into middle of timeline & persistent posting indicators

### ISSUE THREAD

**Issue by custom-jonathan** [urbit/urbit#3987]
**Describe the bug**
When I hit enter to post a message, the grey text will be placed somewhere in the middle of the stream. Sometimes the gray text persists even after they have clearly come through.

**Expected behavior**
Post, it's gray at the bottom until it turns white as the next message at the bottom of the stream.

**Screenshots**
![image](https://user-images.githubusercontent.com/51895363/99843997-124aff80-2b38-11eb-8b06-259cccef5c45.png)

**Desktop (please complete the following information):**
 - OS: Raspbian
 - Browser chromium
 - Base hash of your urbit ship. Run `+trouble` in Dojo to see this.
kb4n

**Smartphone (please complete the following information):**
 - Base hash of your urbit ship. Run `+trouble` in Dojo to see this.

**Additional context**
This started showing up only after I moved my pier to a Raspberry Pi4 with 8GB of ram booting from an SSD. Previously I was on a Digital Ocean Droplet and never saw this behavior.

**Comment by tylershuster** [comment]
This will be fixed upon chat's migration to graph-store (or shortly thereafter)

Basically, the "order" property of the message is not validated, and so while the host of the message may have one order, your client may have another. It results in glitches like this. It's really annoying.

**Comment by custom-jonathan** [comment]
Thanks!

**Comment by matildepark** [comment]
Duplicate of #3847.
